client/src/components/Form/FormTheme.js

```diff
@@ -0,0 +1,94 @@
+import { createTheme } from '@mui/material/styles';
+
+const fontSize = '1.2rem';
+const labelFontSize = '1.3rem';
+const focusedColor = '#28a745';
+
+const theme = createTheme({
+  components: {
+    MuiAutocomplete: {
+      styleOverrides: {
+        root: {
+          label: {
+            fontSize: labelFontSize,
+          },
+          '& label.Mui-focused': {
+            color: focusedColor,
+          },
+          '& .MuiInput-underline:after': {
+            borderBottomColor: focusedColor,
+          },
+        },
+        inputRoot: {
+          fontSize: fontSize,
+        },
+      },
+    },
+
+    MuiFormControl: {
+      styleOverrides: {
+        root: {
+          marginBottom: '1.2rem',
+          label: {
```

Comment:
nit: I see some repeatable elements here. 

Potentially can store in a variable and use in multiple component style overrides
```
const rootBaseStyle = 
{
  label: {
    fontSize: labelFontSize,
  },
  '& label.Mui-focused': {
    color: focusedColor,
  },
  '& .MuiInput-underline:after': {
    borderBottomColor: focusedColor,
  },
  '& .MuiInputBase-input': {
    fontSize: labelFontSize,
  },
}
```

like 
```
MuiFormControl: {
      styleOverrides: {
        root: {
          marginBottom: '1.2rem',
          ...rootBaseStyle
       },
```

Reply:
I didn't think of that, Thanks!
I've made the changes.